ledger/inventory/mod.go

```diff
@@ -0,0 +1,38 @@
+package inventory
+
+import "github.com/golang/protobuf/proto"
+
+// Instance is the smallest unit of storage of a ledger.
+type Instance interface {
+	GetKey() []byte
+	GetValue() proto.Message
+}
+
+// Page represents the state of the inventory at the given point in time.
+type Page interface {
+	GetIndex() uint64
+	GetRoot() []byte
```

Comment:
not sure what is the index and the root of a page

Reply:
comments added